src/components/Shape/LoadingCircle.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { ShapeProps } from './Shape';
+
+export const LoadingCircle: React.FC<ShapeProps> = () => (
```

Comment:
I wouldn't prefix these names with `Loading`. It can be simply `Circle`, `Diamond`, etc.